helm/chart/maesh/values.yaml

```diff
@@ -59,9 +59,18 @@ tracing:
     enabled: true
     localagenthostport: ""
     samplingserverurl: ""
+  datadog:
```

Comment:
Why not just leave it commented out by default instead of uncommented with an `enabled` value of `false`?

Reply:
Sounds good. Will update #370 with the same.